widgets/open-tickets/src/templates/table.ihtml

```diff
@@ -94,23 +94,23 @@
                 </div>
             </td>
             {/if}
```

Comment:
from line 36 to this line, the whole block indentation is wrong and make it difficult to read